lib/Varien/Db/Tree.php

```diff
@@ -223,7 +218,7 @@ public function clear($data = [])
 
         try {
             $this->_db->insert($this->_table, $data);
-        } catch (PDOException $e) {
+        } catch (Zend_Db_Adapter_Exception $e) {
```

Comment:
Can you explain this change? How about line 269, which also catches PDOException?